File: Homeworks/C# Part 1/05.ConditionalStatements/07.SortThreeNumbers/SortThreeNumbers.cs
using System;

class SortThreeNumbers
{
    static void Main()
    {
        short numberA = short.Parse(Console.ReadLine());
        short numberB = short.Parse(Console.ReadLine());
        short numberC = short.Parse(Console.ReadLine());
        short temp;
        if (numberB > numberA && numberB > numberC)
        {
            temp = numberA;
            numberA = numberB;
            numberB = temp;
            if (numberC > numberB)
            {
                temp = numberB;
                numberB = numberC;
                numberC = temp;
            }
        }
        else if (numberC > numberA && numberC > numberB)
        {
            temp = numberC;
            numberC = numberA;
            numberA = temp;
            if (numberC > numberB)
            {
                temp = numberB;
                numberB = numberC;
                numberC = temp;
            }
        }
        else if (numberC > numberB)
        {
            temp = numberB;
            numberB = numberC;
            numberC = temp;
        }
        Console.WriteLine("{0} {1} {2}", numberA, numberB, numberC);
    }
}

File: Homeworks/JS Fundamentals/05. Conditional Statements/04. Sort 3 numbers/sort-three-numbers.js
function solve(args) {
	var a = +args[0];
	var b = +args[1];
	var c = +args[2];
	var temp;
	if (a < b) {
		temp = a;
		a = b;
		b = temp;		
	}
	if (b < c) {
		temp = b;
		b = c;
		c = temp;
	}
	if (a < b) {
		temp = a;
		a = b;
		b = temp;		
	}
	console.log(a + ' ' + b + ' ' + c);
}